tests/issues/99/sap-preconfigure-issue-99-assert.yml

```diff
@@ -0,0 +1,20 @@
+- hosts: all
+  tasks:
+  - name: Check if required packages are installed
+    shell: yum info installed {{ __sap_preconfigure_packages | map('quote') | join(' ') }} | awk '/Name/{n=$NF}/Version/{v=$NF}/Release/{r=$NF}/Description/{printf ("%s\n", n)}'
```

Comment:
Could you use `package_facts` here instead?  https://docs.ansible.com/ansible/2.9/modules/package_facts_module.html#package-facts-module
If not, then could you use `rpm` instead of `yum`, since `yum` is deprecated in el8 in favor of `dnf`, and using `rpm` will more "future proof" the code.

Reply:
I think I once gave up on using package_facts because I failed to handle the output appropriately.
In another piece of code as part of https://github.com/linux-system-roles/sap-preconfigure/issues/104 , I used:
shell: rpm -q --qf "%{NAME}\n" $(echo {{ __sap_preconfigure_packages }} | sed "s.,..g;s,\[,,;s,],,;s,',,g") | grep -v "is not installed"
instead, which worked perfect. As this code is only used as part of a test, are you OK with leaving it in place for now?